src/hjson_decode.cpp

```diff
@@ -13,11 +13,12 @@ struct Parser {
   size_t dataSize;
   int at;
   unsigned char ch;
+  bool parseComments;
```

Comment:
Would be good to keep this internal setting, even if the public parameter is removed. So that the functionality can easily be turned on in the source code.